wasm_test/compiles/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "compiles"
```

Comment:
Couldn't be under `wasm_test` directly and be called `wasm_test`, removing `compiles` as subfolder?

Reply:
i plan to add another test for the persist issue